extension-manifest-v3/src/pages/onboarding/serp.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>More ad blocking available</title>
```

Comment:
is this copy final? shouldn't it be localised? (not that majority of Opera users are in easter europe)

Reply:
Generally, the `<title>` from the head is not visible anywhere when the page is used in the iframe element. I added it just to unify with other HTML files (which also have titles).